src/Misc/tools.ts

```diff
@@ -20,6 +20,8 @@ import { SliceTools } from "./sliceTools";
 declare type Camera = import("../Cameras/camera").Camera;
 declare type Engine = import("../Engines/engine").Engine;
 
+declare const _native: any;

```

Comment:
I added an interface for this. Please use it and add your methods to the interface.

```suggestion
declare const _native: INative;
```